packages/commercetools/theme/static/icons/logo.svg

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
try with this one:

```

944 bytes → 720 bytes 76.27%
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 231 221.4210525"><defs/><rect width="100%" height="100%" fill="none"/><g class="currentLayer"><path fill="#1DCF65" d="M215.745 170.619c.202 2.336-1.64 4.344-3.985 4.344H18.899c-2.339 0-4.179-1.998-3.986-4.33L28.734 3.67C28.906 1.596 30.639 0 32.721 0h164.642c2.076 0 3.807 1.588 3.985 3.656l14.397 166.963z"/><linearGradient id="a" x1=".8002696" x2=".152587" y1=".8600492" y2=".1660911"><stop offset="0" stop-color="#52d67a"/><stop offset="1" stop-color="#5aee87"/></linearGradient><path fill="url(#a)" d="M231.14 214.18c.241 2.914-2.059 5.412-4.982 5.412H5c-2.917 0-5.215-2.486-4.984-5.395L15.893 14.198c.207-2.601 2.376-4.604 4.984-4.604h188.742c2.602 0 4.769 1.995 4.983 4.588L231.14 214.18z"/><g><path fill="#FFF" d="M111.224 156.169c-28.786 0-52.204-28.119-52.204-62.682 0-1.8 1.459-3.26 3.26-3.26s3.26 1.459 3.26 3.26c0 30.968 20.494 56.162 45.685 56.162 25.191 0 45.685-25.194 45.685-56.162 0-1.8 1.459-3.26 3.26-3.26s3.26 1.459 3.26 3.26c-.001 34.563-23.42 62.682-52.206 62.682z"/><path fill="#FFF" d="M143.358 92.613c-.834 0-1.668-.318-2.305-.955-1.273-1.273-1.273-3.337 0-4.61l16.894-16.894c.611-.611 1.44-.955 2.305-.955.865 0 1.694.343 2.305.955l16.729 16.73c1.273 1.273 1.273 3.337 0 4.61-1.273 1.272-3.337 1.273-4.61 0L160.251 77.07l-14.588 14.588c-.637.637-1.471.955-2.305.955z"/></g></g></svg>
```
Optimized version, smaller by 78%

Reply:
thanks, just grabbed it from demo :)